include/stumpless/error.h

```diff
@@ -324,6 +324,35 @@ stumpless_perror( const char *prefix );
 void
 stumpless_set_error_stream( FILE *stream );
 
+/**
```

Comment:
move this function declaration into the `include/private/error.h` header file instead